CONTRIBUTING.md

```diff
@@ -45,22 +45,134 @@ To add a new plugin you need to do the following.
+|----------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------------|
+| **name**                   | The name of the plugin.                                                                                                                                                                                      | Yes      | N/A           |
+| **url**                    | The URL of the plugin's repository.                                                                                                                                                                          | Yes      | N/A           |
+| **callSetup**              | Indicating whether to call the setup function. Useful when `setup` function needs customizations.                                                                                                            | No       | `true`        |
+| **colorscheme**            | The name of the colorscheme.                                                                                                                                                                                 | No       | `name` parameter |
+| **configLocation**         | The location for the Lua configuration.                                                                                                                                                                      | No       | `"extraConfigLuaPre"` if `isColorscheme` then `extraConfigLuaPre`, otherwise `"extraConfigLua"` |
+| **deprecateExtraOptions**  | Indicating whether to deprecate the `extraOptions` attribute. Mainly used for old plugins.                                                                                                                   | No       | `false`       |
+| **description**            | A brief description of the plugin. Can also be used for non-normative documentation, warnings, tips and tricks.                                                                                              | No       | `null`        |
+| **extraConfig**            | Additional configuration for the plugin.                                                                                                                                                                     | No       | `{}`          |
+| **extraOptions**           | Module options for the plugin, to be added _outside_ of the `settings` option. These should be Nixvim-specific options.                                                                                       | No       | `{}`          |
```

Comment:
Slightly out of alignment:

```suggestion
| **extraOptions**           | Module options for the plugin, to be added _outside_ of the `settings` option. These should be Nixvim-specific options.                                                                                      | No       | `{}`          |
```

Reply:
I can't see a problem with the alignment on my end. I'm only on a codespace for this weekend, maybe it's the line wrapping on nvim?

Reply:
Looks like the headings are off too

Reply:
I know this is very, very zoomed out, but I think this is the github diff that f**ked up the rendering. Here's how it is on my end
![image](https://github.com/user-attachments/assets/d0e53e7e-673f-43ff-802a-1e0154c702f5)

Reply:
> I know this is very, very zoomed out, but I think this is the github diff that f**ked up the rendering. Here's how it is on my end ![image](https://private-user-images.githubusercontent.com/28595242/386706925-d0e53e7e-673f-43ff-802a-1e0154c702f5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE2OTMwMTUsIm5iZiI6MTczMTY5MjcxNSwicGF0aCI6Ii8yODU5NTI0Mi8zODY3MDY5MjUtZDBlNTNlN2UtNjczZi00M2ZmLTgwMmEtMWUwMTU0YzcwMmY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE1VDE3NDUxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEzYWQyOThiMWJkMzlkODk4NmMxZGFmMjk0NzhjZTMyZDBkYWYwMGFmNDJkYzg0ZTJmNzVhNTMxMzY2NjNlZjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.XrNwTaqCGWvl3NppfDfrs_qV0vJkzixJAlTwqBHMEjE)

Actually, this in not the first time Github does this... I've had some rendering issue with the diff on some projects with whitespaces and newlines.